docs/en/trust.rst

```diff
@@ -747,6 +724,198 @@ Each JWT containing a Trust Chain in the JWT headers can be verified over time,
+- **Constraints in the Chain**: constraints can be applied within the Trust Chain regarding critical aspects such as the delegation of trust, the number of intermediates, and the domains involved.
+- **Public Key Distribution**: Both systems involve the distribution of the public key of the Trust Anchor to ensure entities can verify the trust chain.
+- **Registry of Expired Keys**: Maintaining a registry of expired keys is crucial for both, ensuring non-repudiation of past signatures even when keys change.
+
+
+Federation Trust Anchor and X.509 CA
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the context of OpenID Federation, the Trust Anchor plays a role similar to that of a Certificate Authority (CA) in X.509-based Public Key Infrastructures (PKIs). Both serve as foundational elements of trust within their respective systems. In this document, the term "Trust Anchor" is often used to encompass both concepts. The trust infrastructure described here aligns the OpenID Federation Trust Anchor with the X.509 PKI Certificate Authority, making therefore them a single unique entity supporting both `RFC5280`_ and OpenID Federation 1.0.
+
+X.509 Certificates Issuance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In an OpenID Federation, each participant is required to self-issue its Entity Configuration, signing it with one of its cryptographic keys that are attested by Immediate Superiors.
+
+In the same way, each federation Entity has the autonomy to issue a signed statement about itself in the form of a X.509 Certificate.
+Federation participants that need to issue X.509 Certificates about themselves and for their specific purposes, can issue and sign X.509 Certificates using one of their Federation Entity Keys attested by their Federation Authorities (Immediate Superior). This process aligns the issuance of X.509 Certificates with the federation's delegation paradigm.
+
+This is feasible because the X.509 Certificate can be verified using a X.509 Certificate Chain, similar to the approach used for Entity Configurations in OpenID Federation.
+
+Federation Leaves are not Certificate Authorities (CAs) or CA intermediaries authorized to issue X.509 certificates for their subordinates. Instead, Federation Leaves act as intermediaries for issuing certificates solely about themselves. This is accomplished by applying appropriate naming constraints to ensure that X.509 certificates are correctly scoped.
+Naming constraints are applied by Immediate Superiors within the certificates issued to the Leaf entity, specifically concerning the Leaf's Federation Entity Keys. As a result, the Leaf can only issue X.509 certificates about itself, thereby maintaining the integrity of the Trust Chain.
+
+.. name-constraints::
+   :permitted: URI.1=http://leaf.example.com
+   :permitted: CN=leaf.example.com
+
+When a participant self-issues an X.509 Certificate, it adheres to the following requirements:
```

Comment:
It may be worthwhile identifying if there are an embargoed/forbidden values here.  Ones that come to mind are: localhost, localhost.localdomain, localhost.domain, the IETF restricted example.com / example.org domains, 127.0.0.1 and what about names without dots? Are they valid? what about the strings that look like wildcards but are not? *.example.org -- are wildcards forbidden?  Is there an obligation that the certificate CN / SAN DNS must persist only at issuance or into perpetuity? Think of how infrequent the intermediates will 'roll' or may reference and if the DNS name is lost or overtaken by someone else, does THAT pose a risk to the trust chain?  Thoughts welcome.

Reply:
yeah, it would be something like the one below:

````
.. name-constraints::
   :permitted: URI.1=https://leaf.example.com
   :permitted: CN=leaf.example.com
   :permitted: DNS=leaf.example.com
   :excluded: DNS=localhost
   :excluded: DNS=localhost.localdomain
   :excluded: DNS=127.0.0.1
   :excluded: DNS=example.com
   :excluded: DNS=example.org
   :excluded: DNS=example.net
   :excluded: DNS=*.example.org
````

I want also to add that in this context, wildcards cannot be allowed for Leaf's CA, since the Leaf can only issue certificates about their entity id hostname and uri.